src/main/java/com/greenfox/lvlup/model/entity/Pitch.java

```diff
@@ -26,7 +25,7 @@
   @ManyToOne
   private User user;
   @ManyToOne
-  private BadgeLevel badgeLevel;
```

Comment:
could you rename this BadgeLevel to match the Dev version please (badgeLevel instead of level)